en/docs/administration/configure-custom-domain.md

```diff
@@ -0,0 +1,27 @@
+# Configure Custom Domain
+
+A custom domain is essential for proper branding, discoverability, and credibility of your site. Choreo allows you to configure a custom domain for your organization's developer portal.
+
+Follow the steps below to configure the custom domain for the developer portal:
+
+## Prerequisites
+
+1. Sign in to the Choreo Console at <a>https://console.choreo.dev/</a> using a Google/ GitHub/ Microsoft account.
+2. If you are a new user, create an organization. Enter a unique organization name and create an organization. For example, "Stark Industries".
```

Comment:
We are repeating "create an organization". Should we go with:
If you are a new user, enter a unique organization name and create an organization. For example, "Stark Industries".

Reply:
done